lib/src/main/java/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -181,28 +192,28 @@ class StatefulLayout : FrameLayout, StateContainer<Int, State> {
      * @throws [NoSuchElementException] if [id] was not found.
      */
```

Comment:
I'm not a javadoc pro so i may be wrong. Should you not add the fact that we are using in this signature `areTransitionsEnabled` class parameter ?